### PR TITLE
add headers

### PR DESCRIPTION
**Pull request by JoseMotto-sg**
## Change description

Repository header update. Note that this repo doesn't have terraform-docs, so only header is updated

## Type of change
- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)

## Related issues

> Fix [#1](No Applicable) 

### Development

- [x] Lint rules pass locally
- [x] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### Code review 

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [x] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Review by fernandoPedraza-sg — CHANGES_REQUESTED** [review]
@JoseMotto-sg, the header in the README.md file still needs to be updated. It must be placed directly in the README.md file in the root directory.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+<p align="center">                                                                                                                                            ─╯
```
Comment (fernandoPedraza-sg) [inline README.md]: @JoseMotto-sg, please remove spaces and text character at the end of this line.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+  <img src="https://github.com/StratusGrid/terraform-readme-template/blob/main/header/stratusgrid-logo-smaller.jpg?raw=true" />
+  <p align="center">                                                           
+    <a href="https://stratusgrid.com/book-a-consultation">Contact Us Test</a> |                  
+    <a href="https://stratusgrid.com/cloud-cost-optimization-dashboard">Stratusphere FinOps</a> |
+    <a href="https://stratusgrid.com">StratusGrid Home</a> |
+    <a href="https://stratusgrid.com/blog">Blog</a>
+  </p>                    
+</p>
 # workflow-config
```
Comment (fernandoPedraza-sg) [inline README.md:12]: @JoseMotto-sg, note that an additional space is required here to maintain the header format:

File: .config/header.md
Comment (fernandoPedraza-sg) [inline .config/header.md:1]: @JoseMotto-sg, please consider removing this file, and folder, as it si not used for the README header.

**Review by fernandoPedraza-sg — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (fernandoPedraza-sg) [inline README.md:12]:
```suggestion

# workflow-config
```
@JoseMotto-sg, note that an additional space is required here to maintain the header format:

File: README.md (hunk 1 above)
Comment (JoseMotto-sg) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (JoseMotto-sg) [reply README.md:12]: done

File: .config/header.md
Comment (JoseMotto-sg) [reply .config/header.md:1]: done

**Review by fernandoPedraza-sg — APPROVED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -1,3 +1,14 @@
+<p align="center">
+  <img src="https://github.com/StratusGrid/terraform-readme-template/blob/main/header/stratusgrid-logo-smaller.jpg?raw=true" />
+
+  <p align="center">                                                           
+    <a href="https://stratusgrid.com/book-a-consultation">Contact Us Test</a>
```
Comment (jheison-rodriguez) [inline README.md:5]: remove ` Test`

File: README.md [hunk 4]
```diff
@@ -1,3 +1,14 @@
...
+    <a href="https://stratusgrid.com/cloud-cost-optimization-dashboard">Stratusphere FinOps</a>
+    <a href="https://stratusgrid.com">StratusGrid Home</a>
```
Comment (jheison-rodriguez) [inline README.md:7]: there is missing vertical line `|`at the end

File: README.md [hunk 5]
```diff
@@ -1,3 +1,14 @@
+<p align="center">
+  <img src="https://github.com/StratusGrid/terraform-readme-template/blob/main/header/stratusgrid-logo-smaller.jpg?raw=true" />
+
+  <p align="center">                                                           
+    <a href="https://stratusgrid.com/book-a-consultation">Contact Us Test</a>
+    <a href="https://stratusgrid.com/cloud-cost-optimization-dashboard">Stratusphere FinOps</a>
```
Comment (jheison-rodriguez) [inline README.md:6]: there is missing vertical line `|`at the end

File: README.md (hunk 3 above)
Comment (jheison-rodriguez) [inline README.md:5]: there is missing vertical line `|`at the end

**Review by jheison-rodriguez — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (JoseMotto-sg) [reply README.md:5]: Please open a suggestion directly

File: README.md (hunk 4 above)
Comment (JoseMotto-sg) [reply README.md:7]: Please open a suggestion directly

File: README.md (hunk 5 above)
Comment (JoseMotto-sg) [reply README.md:6]: Please open a suggestion directly

File: README.md (hunk 3 above)
Comment (JoseMotto-sg) [reply README.md:5]: Please open a suggestion directly

**Pull request closed without merging**